src/index.html

```diff
@@ -4,6 +4,11 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>Virtual CT Board</title>
+    <link
```

Comment:
Should we include this in the bundle?

Reply:
I guess you mean the CDN link? I know this is a bit ugly at the moment but I could not figure out how to include this in the rollup build. In order to show the symbols, font files are required which are references relatively in the parent folder in a folder called "fonts". Unfortunately rollup does not support the processing of font files imported via import statements, the build will fail.